src/main/app/src/index.tsx

```diff
@@ -74,43 +71,22 @@ window.onerror = (errorMsg, _url, line, col, errorObj) => {
         uninterestingErrors[errorKey] = true;
       }
     };
-    StackTrace.fromError(errorObj)
-      .then((err) => {
-        console.log(err);
-        send(JSON.stringify(err));
-      })
-      .catch(() => {
-        send(JSON.stringify(errorObj));
-      });
-  }
-};
-
-const useSyncAppPage = () => {
-  const dispatch = useDispatch();
-  const location = useLocation();
-
-  useEffect(() => {
-    dispatch(locationChanged(location));
-  }, [dispatch, location]);
-};
-
-const InitGate = ({ children }) => {
-  const { status: urlStatus } = useQueryOnce('urls', configureUrls);
-  const { status: i18nStatus } = useQueryOnce('i18n', configureI18n);
-  const { status: configurationStatus } = useQueryOnce('configuration', getConfiguration);
-
-  useSyncAppPage();
-
-  if ([urlStatus, i18nStatus, configurationStatus].includes('loading')) {
-    return <LoadingCircle />;
-  } else if ([urlStatus, i18nStatus, configurationStatus].includes('error')) {
-    return <div>Sovelluksen lataaminen epäonnistui!</div>;
-  } else {
-    return children;
+    if (errorObj) {
+      StackTrace.fromError(errorObj)
+        .then((err) => {
+          console.error(err);
+          send(JSON.stringify(err));
+        })
+        .catch(() => {
+          send(JSON.stringify(errorObj));
+        });
+    } else {
+      send();
+    }
   }
 };
 
-const root = createRoot(document.getElementById('wrapper'));
+const root = createRoot(document.getElementById('wrapper') as Element);
```

Comment:
Tässäkin voisi olla lyhyt kommentti as:in käytöstä (vaikka onkin tässä aika itsestään selvä)